equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileServiceTest.java

```diff
@@ -38,25 +41,66 @@ class CrmFileServiceTest {
     @Autowired
     CrmFileService crmFileService;
 
+    Map<Long, String> crmFileTests;
+    Map<Long, Integer> crmFileTestTypes;
+    Map<Long, Integer> linkedAttachmentTests;
+
 
     @BeforeAll
     void preTest() throws IOException {
-        // Mocking good XML
-        FileInputStream fis = new FileInputStream("src/test/resources/Crm5MockFile_5001604.txt");
-        JSONObject mockedCrm5Json = new JSONObject(IOUtils.toString(fis, StandardCharsets.UTF_8));
-        byte[] fileDataByte = XML.toString(mockedCrm5Json).getBytes(StandardCharsets.UTF_8);
-        createMock(10L, fileDataByte);
-
-        // Mocking XML with bad JSON format
-        fis = new FileInputStream("src/test/resources/Crm5MockOFDFile_WrongFormat.txt");
-        fileDataByte = fis.readAllBytes();
-        createMock(11L, fileDataByte);
+        crmFileTestTypes = new HashMap<>();
+        crmFileTestTypes.put(5001912L, CRM_TYPE_4);
+        crmFileTestTypes.put(5001604L, CRM_TYPE_5);
+        crmFileTestTypes.put(11L, CRM_TYPE_5);
+        crmFileTestTypes.put(5001662L, CRM_TYPE_7);
+        crmFileTestTypes.put(4808706L, CRM_TYPE_7);
+        crmFileTestTypes.put(5001597L, CRM_TYPE_7);
+
+
+        crmFileTests =  new HashMap<>();
+        crmFileTests.put(5001604L, "src/test/resources/Crm5MockFile_5001604.txt"); // Mocking good XML
```

Comment:
`src/test/resources` - Do you we need to give the full path?

Reply:
I think using full path makes it clear which file it refers. Besides,  if it were looking for relative location,  then it wouldn't find the files